packages/oats-runtime/package.json

```diff
@@ -23,6 +23,10 @@
   },
   "dependencies": {
     "@smartlyio/safe-navigation": "^5.0.1",
+    "@types/encodeurl": "^1.0.0",
+    "@types/escape-html": "^1.0.1",
+    "encodeurl": "^1.0.2",
```

Comment:
could we do with just https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURI instead of adding a dependency?

Reply:
oh I didn't know about it, but i'll fix this